scipy/interpolate/_ppoly.pyx

```diff
@@ -458,6 +458,11 @@ def real_roots(double[:,:,::1] c, double[::1] x, double y, bint report_discont,
 
     wr = <double*>libc.stdlib.malloc(c.shape[0] * sizeof(double))
     wi = <double*>libc.stdlib.malloc(c.shape[0] * sizeof(double))
+    if not wr or not wi:
+        libc.stdlib.free(wr)
```

Comment:
This one should read `wi` I think.

Reply:
Thank you for reading. My eyes just glazed over this as I was reviewing.